packages/goto/scripts/postinstall.js

```diff
@@ -1,54 +1,116 @@
 'use strict'
 
 const { PuppeteerBlocker, getLinesWithFilters } = require('@cliqz/adblocker')
+const debug = require('debug-logfmt')('browserless:goto:postinstall')
 const { promisify } = require('util')
+const split = require('binary-split')
+const crypto = require('crypto')
 const { EOL } = require('os')
 const got = require('got')
 const fs = require('fs')
 
 const writeFile = promisify(fs.writeFile)
 
 // uBlock Origin –  https://github.com/uBlockOrigin/uAssets/tree/master/filters
-const RESOURCES = 'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/resources.txt'
+const RESOURCES =
+  'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/resources.txt'
 
 const FILTERS = [
   // uBlock Origin –  https://github.com/uBlockOrigin/uAssets/tree/master/filters
   'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/filters.txt',
+  'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/annoyances.txt',
   'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/badware.txt',
   'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/privacy.txt',
   'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/resource-abuse.txt',
   'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/unbreak.txt',
-  'https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/annoyances.txt',
-
-  // Easylist – https://easylist.to/
-  'https://easylist.to/easylist/easylist.txt',
-  'https://easylist.to/easylist/easyprivacy.txt',
-  'https://easylist.to/easylist/fanboy-annoyance.txt',
-  'https://easylist.to/easylist/fanboy-social.txt',
 
+  // Fanboy – https://www.fanboy.co.nz
+  // Easylist, Easyprivacy, Enhanced Trackers
+  'https://www.fanboy.co.nz/r/fanboy-complete.txt',
+  // Fanboy Annoyances List + Fanboy-Social List
+  'https://easylist-downloads.adblockplus.org/fanboy-annoyance.txt',
   // Other
-  'http://pgl.yoyo.org/as/serverlist.php?hostformat=adblockplus;showintro=0&mimetype=plaintext',
-  'https://fanboy.co.nz/fanboy-cookiemonster.txt',
-  'https://raw.githubusercontent.com/liamja/Prebake/master/obtrusive.txt'
+  // Cookies Lists
+  'https://www.fanboy.co.nz/fanboy-cookiemonster.txt',
+  // crypto miners
+  'https://raw.githubusercontent.com/hoshsadiq/adblock-nocoin-list/master/nocoin.txt'
+]
+
+const HOSTS = [
+  // Steven Black hosts – https://github.com/StevenBlack/hosts
+  'https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts'
```

Comment:
Just a side-note. There used to be support for these in `@cliqz/adblocker` at some point but was removed because usually there is an alternative with Easylist syntax. We could consider adding it back if it's more convenient.